docs/PokemonCreation.md

```diff
@@ -0,0 +1,68 @@
+## Pokemon Creation
+
+Pokemon creation in FullScreenPokemon consists of three different types, Unit Test Pokemon creation, Trainer Battle Pokemon creation, and Wild Pokemon Grass Encounter creation.
+The two main files that handle Pokemon creation are [`Equations`](../src/components/Equations.ts) and [`Utilities`](../src/components/Utilities.ts),both of which incorporate ['numberMaker'](../node_modules/numbermakr/lib) for random generation of numbers.
+[`Equations`](../src/components/Equations.ts) includes functions for choosing random wild Pokemon and for creating the actual Pokemon object. [`Utilities`](../src/components/Utilities.ts) facilitates the creating of a wild Pokemon by preparing parameters than calling [`Equations`](../src/components/Equations.ts) creation of the Pokemon object.
+
+A newly created Pokemon has members of types:
```

Comment:
Manually listing out the types here is fragile: if they ever change (which is very likely), this place will become out-of-sync. Instead just link to the file they're in and say their interface name. We can't link to line number because that'll probably change too.